libraries/AP_Terrain/AP_Terrain.cpp

```diff
@@ -67,6 +67,14 @@ const AP_Param::GroupInfo AP_Terrain::var_info[] = {
     // @Range: 0.05 50000
     // @User: Advanced
     AP_GROUPINFO("MARGIN",   3, AP_Terrain, margin, 0.05),
+
+    // @Param: OFS_MAX
+    // @DisplayName: Maximum reference offset
```

Comment:
Could we change the display name to "Terrain reference offset maximum"?

Reply:
done